jekyll/_cci2/deployment-examples.md

```diff
@@ -0,0 +1,653 @@
+---
+layout: classic-docs
+title: "Deployment Examples"
+short-title: "Deployment Examples"
+---
+This document presents example config for a variately of popular deployment targets. Many of the examples use orbs: CircleCI and partners have developed a catalogue of orbs that enable you to quickly deploy applications with minimal config. Details of all orbs can be found in the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/).
+
+* TOC
+{:toc}
+
+#### Notes on Examples
+{:.no_toc}
+
+* In order to use orbs you must use `version 2.1` config, and enable pipelines for your project. 
```

Comment:
I've just merged the pipelines snippet if you want to use that here!

Reply:
🎉

Reply:
Just realised I would have to rebase again so I will make this change after this PR gets merged I think 😅